lstchain/data/lstchain_standard_config.json

```diff
@@ -90,70 +85,95 @@
     "reco_disp_dx",
     "reco_disp_dy"
   ],
-
-  "allowed_tels": [1, 2, 3, 4],
+  "allowed_tels": [
+    1,
+    2,
+    3,
+    4
+  ],
   "max_events": null,
   "custom_calibration": false,
   "write_pe_image": false,
   "image_extractor": "LocalPeakWindowSum",
-  "image_extractor_for_muons": "GlobalPeakWindowSum",
-  "image_extractor_config": {},
+  "image_extractor_config": {
+    "LocalPeakWindowSum": {
+      "window_shift": 4,
+      "window_width": 8
+    },
+    "GlobalPeakWindowSum": {
+      "window_shift": 4,
+      "window_width": 8
+    }
+  },
   "gain_selector": "ThresholdGainSelector",
   "gain_selector_config": {
-    "threshold":  3500
+    "threshold": 3500
   },
-
-  "LocalPeakWindowSum":{
-     "window_shift": 4,
-     "window_width":8
-    },
-  "GlobalPeakWindowSum":{
-     "window_shift": 4,
-     "window_width":8
+  "image_extractor_for_muons": "GlobalPeakWindowSum",
+  "image_extractor_for_muons_config": {
```

Comment:
Same as above. Why not only give the config for the `GlobalPeakWindowSum`?

Reply:
For this PR I'd like to simply fix the config bug.
The whole config will need to be changed and improved with the use of tools and ctapipe0.8.